WhlgPortalWebsite.Data/DataAccessProvider.cs

```diff
@@ -161,4 +173,38 @@ public async Task<bool> IsEmailAddressInUseAsync(string emailAddress)
             StringComparison.CurrentCultureIgnoreCase
         ));
     }
+
+    public async Task<IEnumerable<LocalAuthority>> GetAllLasAsync()
+    {
+        return await context.LocalAuthorities.ToListAsync();
+    }
+
+    public async Task<IEnumerable<Consortium>> GetAllConsortiaAsync()
+    {
+        return await context.Consortia.ToListAsync();
+    }
+
+    public async Task AddLaToDeliveryPartnerAsync(User user, LocalAuthority localAuthority)
+    {
+        user.LocalAuthorities.Add(localAuthority);
+        await context.SaveChangesAsync();
+    }
+
+    public async Task AddConsortiumToDeliveryPartnerAsync(User user, Consortium consortium)
+    {
+        user.Consortia.Add(consortium);
+        await context.SaveChangesAsync();
+    }
```

Comment:
Here we add LA / consortium to the user - do we also need to add the user to the LA / consortium record?
I see both LocalAuthority and Consortium have a `Users` property that is a list of users, but I'm unsure what that is used for as I can't see it used in the app at the moment.

Reply:
EF seems to be able to handle this many to many relation between LAs & Users, so adding to `Consortia` here will add to the `Users` property of a Consortia & vice versa